contributors.txt

```diff
@@ -6,4 +6,5 @@ Jane Ansell
 Neil Heraghty
 Tom Roberts
 Lucrezia Cester
+Siân Culley
```

Comment:
welcome Sian!

Reply:
@superresolusian – thank you, thank you.